src/provider/neoWorkTransformer.js

```diff
@@ -105,19 +105,18 @@ export function workResponse(response, context, state) { // eslint-disable-line
       // But if you just do 'let x = Array(3).fill({})', you will get an array with the same
       // object three times. So if you add to x[0], you will also add to x[1] and x[2].
       // Not exactly what I expected.
-      if (resp.data) {
-        envelope.data = Array(resp.data.length).fill(1).map(x => { // eslint-disable-line no-unused-vars
+      if (resp.data || resp[0].data) {
+        // Length is dependent on which services were called.
```

Comment:
Isn't the length of the result always the number of pids passed to the service?
Such that each entry of the result matchs each of the pids in the query.